src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -192,10 +197,10 @@ function MoneyTemporaryForRefactorRequestParticipantsSelector({participants, onF
             ];
 
             if (iouType === CONST.IOU.TYPE.INVOICE) {
-                const primaryPolicy = Policy.getPrimaryPolicy(activePolicyID);
+                const primaryPolicy = Policy.getPrimaryPolicy(activePolicyID ?? undefined);
```

Comment:
You can revert this change and instead adjust getPrimaryPolicy function:

```ts
function getPrimaryPolicy(activePolicyID?: OnyxEntry<string>): Policy | undefined {
    const activeAdminWorkspaces = PolicyUtils.getActiveAdminWorkspaces(allPolicies);
    const primaryPolicy = allPolicies?.[activePolicyID ?? ''];

    return primaryPolicy ?? activeAdminWorkspaces[0];
}
```

```suggestion
                const primaryPolicy = Policy.getPrimaryPolicy(activePolicyID);
```

Reply:
updated